### ISSUE TITLE
suggestion: notification placement

### ISSUE THREAD

**Issue by hire-vladimir** [inket/CustomNC#2]
would be amazing if ability was added to this project that would allow changing the placement of the notification, i.e., appear on bottom right corner, instead of the default top. Much of the time notification cover up buttons or tabs or the applications, causing frustration.

**Comment by ylluminate** [comment]
Yes, I agree, that this would be fantastic.  Aside from simply the location on a single display, it would be so very useful to control WHICH display it appears on.  I have multiple displays and using the main has just never been good for me.

**Comment by SainteCelery** [comment]
Is this possible? The upper right corner is just about the worst possible place to have a banner sit.
